recipes/sankee/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sankee" %}
+{% set version = "0.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sankee-{{ version }}.tar.gz
+  sha256: 613db2e63cdc51e5358f590a7836aac8d5a7bbfc39da5c9138a38e4badd6d87a
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - earthengine-api >=0.1.230
+    - ipykernel
+    - nbformat >=4.2.0
+    - numpy
+    - pandas
+    - plotly
+    - python
+    - pywin32  # [win]
```

Comment:
I see upstream lists this, but it doesn't seem to be directly imported anywhere. Any idea if it's *really* needed? If not, this could be made `noarch: python`

Reply:
You're right, that's not needed. Good catch! I've removed it and specified noarch.